### PR TITLE
[MOD-175][Hotfix] Include api version in queryHasMany request

### PR DESCRIPTION
**Pull request by laurenbarker**
## Purpose
Prevent timezone issues.

## Changes
* Included header with api version in queryHasMany request.
    * Used version 2.5 because I get errors with 2.6
* Updated accessing the metadata since it changed between API version.

## Side effects
None expected.

### CODE REVIEW

File: app/services/store.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ export default Store.extend({
             if (url) {
                 $.ajax(url, {
                     data: queryParams,
+                    headers: {
+                        ACCEPT: 'application/vnd.api+json; version=2.5',
```
Comment (aaxelb) [inline app/services/store.js:32]: Could you import `ember-osf/adapters/osf-adapter` and use its `headers` instead?

**Review by aaxelb — CHANGES_REQUESTED** [review]
Out of curiosity, what errors did you get with 2.6?

**Comment by laurenbarker** [comment]
500 from the API:
```
TransactionManagementError: An error occurred in the current transaction. You can't execute queries until the end of the 'atomic' block.
[01/Nov/2017 15:23:55] "GET /v2/preprint_providers/osf/preprints/?filter%5Breviews_state%5D=accepted&filter%5Bnode_is_public%5D=true&meta%5Breviews_state_counts%5D=true&sort=-date_last_transitioned&page=1 HTTP/1.1" 500 59
```

File: app/services/store.js (hunk 1 above)
Comment (laurenbarker) [reply app/services/store.js:32]: Tried and failed to import the headers as discussed on flowdock 😞 . Added ticket and comment.

**Review by aaxelb — APPROVED** [review]

**Pull request merged**